app/chat/src/ChatServer.kt

```diff
@@ -69,22 +69,16 @@ class ChatServer {
     }
 
     private suspend fun broadcast(message: String) {
```

Comment:
Was this working, or do I miss something? When tested, it, the second client was receiving empty messages.